service-fabric-cluster-5-node-1-nodetype/azuredeploy.json

```diff
@@ -0,0 +1,592 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterLocation": {
+      "type": "string",
+      "allowedValues": [
+        "westus",
+        "eastus"
+      ],
+      "metadata": {
+        "description": "Location of the Cluster"
+      }
+    },
+    "clusterName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of your cluster - Between 3 and 23 characters.  Lowercase letters and numbers"
```

Comment:
Will the deployment fail if a user enters a capital letter? If so, then I suggest creating a variable that converts the value in the parameter here to all lower case using the toLower function